src/contents/all/components/app/App.tsx

```diff
@@ -115,14 +115,20 @@ const App = () => {
     }
 
     if (request.message === "clicked_browser_action") {
-      if (user) {
-        setTimeout(() => {
+      chrome.storage.local.get("analogueJWT", function(token) {
+        if (!user && Object.keys(token).length !== 0) {
+          agent.setToken(token.analogueJWT)
+          //dispatch({ type: 'SET_USER', user: **GET USER FROM TOKEN** })
```

Comment:
@joelrojo This is the last missing piece. Need a way to get and dispatch User from a stored token. was thinking agent.Auth.current() but i don't think that does it. Any ideas?